Detector/DetFCChhTrackerTkLayout/src/TkLayoutEndcap_Geo.cpp

```diff
@@ -126,6 +134,14 @@ static DD4hep::Geometry::Ref_t createTkLayoutTrackerEndcap(DD4hep::Geometry::LCD
   PlacedVolume placedDiscVolume = envelopeVolume.placeVolume(discVolume, DD4hep::Geometry::Position(0, 0, currentZ));
   placedDiscVolume.addPhysVolID("disc", discCounter);
   ++discCounter;
+  Acts::ActsExtension::Config layConfig;
+  // the local coordinate systems of modules in dd4hep and acts differ
+  // see http://acts.web.cern.ch/ACTS/latest/doc/group__DD4hepPlugins.html
+  layConfig.axes = "XZY"; // correct translation of local x axis in dd4hep to local x axis in acts
```

Comment:
This is because end caps and barrels use different shapes. For the box intuitively the translation xy-> xy and z->thickness was chosen. 
For the [TGeoTrd2](https://root.cern.ch/doc/v608/classTGeoTrd2.html) in Root the basic sides are x and y while z is the stable length. Therefore the projection needs to the 2D trapezoid needs to be xz->xy while y is the thickness. 
I should probably upload some drawings to make clear which orientation ACTS is using. We can also discuss if using xz->xy for the box volume for consistency would be better.